src/icons/general/IconShare.js

```diff
@@ -14,29 +14,22 @@ type Props = {
     width?: number,
 };
 
-const IconShare = ({ className = '', color = BOX_BLUE, height = 26, title, width = 26 }: Props) => (
+const IconShare = ({ className = '', color = boxBlue, height = 32, title, width = 32 }: Props) => (
     <AccessibleSVG
         className={`icon-share ${className}`}
```

Comment:
since this PR is already a breaking change, might at well update classes to `bdl-IconXYZ`

Reply:
Done